tools/ovc/openvino/tools/ovc/moc_frontend/pipeline.py

```diff
@@ -39,9 +39,26 @@ def get_enabled_and_disabled_transforms():
 
 def raise_exception_for_input_output_cut(model_inputs_or_outputs: List[Place], new_nodes: List[dict], is_input: bool):
     for new_node in new_nodes:
-        node = new_node['node']
+        node_found = False
+        for item in model_inputs_or_outputs:
+            if is_input:
+                name = new_node['input_name']
+                if name in item.get_names():
+                    node_found = True
+                    break
+                elif name + ":0" in item.get_names():
+                    node_found = True
+                    break
+            else:
+                name = new_node['output_name']
+                if name in item.get_names():
+                    node_found = True
+                    break
+                elif name + ":0" in item.get_names():
+                    node_found = True
+                    break
 
-        if not any([item.is_equal(node) for item in model_inputs_or_outputs]):
+        if not node_found:
```

Comment:
Not sure about these changes. Looks not needed

Reply:
This change is needed to prevent ovc failing if user sets operation name in `input` parameter. This method checks that input names provided by user are among `InputModel` inputs, it is needed to forbid model cutting. But `InputModel` has inputs with <op name>:0, so if we set `input=op_name`, `ovc` fails with error, that `op_name` is not found among model inputs.

 So I changed this method to check not only exact name among input names, but also name:0.

Reply:
Removed this change as greed.